.travis.yml

```diff
@@ -1,10 +1,11 @@
+dist: bionic
 sudo: required
 language: ruby
 services:
   - docker
 cache: bundler
 rvm:
-- 2.5.3
+- 2.6.3
```

Comment:
Is this the Ruby version or the RVM version? Should we delete the `.ruby-version` file from the repo since this will override that file?

Reply:
This is the ruby version [AFAIK](https://docs.travis-ci.com/user/languages/ruby/#specifying-ruby-versions-and-implementations). I didn't realize we had a .ruby-version file, we should use that because Travis will read it and we can also support developer clients that way too.